src/gwt/src/org/rstudio/studio/client/workbench/views/output/find/FindInFilesDialog.java

```diff
@@ -92,6 +101,37 @@ public native final boolean isCaseSensitive() /*-{
       private native JsArrayString getFilePatternsNative() /*-{
          return this.filePatterns;
       }-*/;
+
+      public native final void updateResultsCount(int count) /*-{
+         this.resultsCount += count;
+      }-*/;
+
+      public native final int getResultsCount() /*-{
+         return this.resultsCount;
+      }-*/;
+
+      public native final void clearResultsCount() /*-{
+         this.resultsCount = 0;
+      }-*/;
+
+      public native final void updateErrorCount(int count) /*-{
+         this.errorCount += count;
+      }-*/;
+
+      public native final int getErrorCount() /*-{
+         return this.errorCount;
+      }-*/;
+
+      public native final void updateReplaceErrors(String errors) /*-{
+         if (this.replaceErrors)
```

Comment:
JavaScript's type coercion can make this weird, e.g. the string "0" is considered to be false. Would recommend checking for typeof string here instead.

Reply:
Fixed